test/unit/specs/validators/sameAs.spec.js

```diff
@@ -9,6 +9,9 @@ describe('sameAs validator', () => {
   it('should not validate different string', () => {
     expect(sameAs('first')('world', parentVm)).to.be.false
     expect(sameAs('second')('hello', parentVm)).to.be.false
+    expect(sameAs('first')(undefined, parentVm)).to.be.false
+    expect(sameAs('first')(null, parentVm)).to.be.false
```

Comment:
Thank you for responsing.
I think that `should not validate different value` rather than should `should not validate different string` is appropriate.